poclu/misc.c

```diff
@@ -556,7 +556,7 @@ poclu_load_program_multidev (cl_context context, cl_device_id *devices,
     {
       TEST_ASSERT (device != NULL);
       strcat (final_opts, " -x spir -spir-std=1.2");
-      err = clGetDeviceInfo (device, CL_DEVICE_EXTENSIONS, 1024, extensions,
+      err = clGetDeviceInfo (device, CL_DEVICE_EXTENSIONS, 10000, extensions,
```

Comment:
This is getting a bit uncomfortably large for a stack-allocated array with an arbitrary hardcoded size. Maybe query the actual size & allocate dynamically?